src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -47,10 +48,10 @@ public DriveTrain() {
     this.m_backRight = new VictorSPX(RobotMap.m_motorBackRight);
 
     /* Invert only Left side so Hatch side is front */
-    this.m_frontLeft.setInverted(true);
-    this.m_backLeft.setInverted(true);
-    this.m_frontRight.setInverted(false);
-    this.m_backRight.setInverted(false);
+    this.m_frontLeft.setInverted(false);
```

Comment:
Since your code is very old off of master, this inversion may not be necessary or already existing in the code